.repo-metadata.json

```diff
@@ -6,6 +6,7 @@
   "issue_tracker": "https://issuetracker.google.com/issues/new?component=187134&template=0",
   "release_level": "alpha",
   "language": "python",
+  "library_type": "REST",
```

Comment:
I believe this a GAPIC, since it is generated by the python generator.

Reply:
Done.